src/software/thunderscope/constants.py

```diff
@@ -336,6 +336,11 @@ class TrailValues:
     DEFAULT_TRAIL_SAMPLING_RATE = 0
 
 
+class ProtoConfigurationConstant:
+    DEFAULT_SAVE_DIRECTORY = "/tmp/tbotspython/thunderbots_configurations_proto"
```

Comment:
nit:
```suggestion
    DEFAULT_SAVE_DIRECTORY = "/tmp/tbotspython/thunderbots_configuration_proto"
```

Reply:
Since it looks like we do `chown -R` in `setup_software.sh`, let's also change this to `/opt`
```suggestion
    DEFAULT_SAVE_DIRECTORY = "/opt/tbotspython/thunderbots_configuration_proto"
```